en/prerequisites.md

```diff
@@ -16,212 +16,219 @@ This document introduces the hardware and software prerequisites for deploying a
 
     > **Note:**
     >
     > `LimitNOFILE` must be explicitly set to `1048576` or a greater value, other than `infinity` by default. Due to [a bug of `systemd`](https://github.com/systemd/systemd/commit/6385cb31ef443be3e0d6da5ea62a267a49174688#diff-108b33cf1bd0765d116dd401376ca356L1186), the `infinity` value in some versions of `systemd` is `65536`.
 
-## Hardware and deployment requirements
+## Kubernetes service
 
-+ 64-bit generic hardware server platform in the Intel x86-64 architecture and 10 Gigabit NIC (network interface card), which are the same as the server requirements for deploying a TiDB cluster using binary. For details, refer to [Hardware recommendations](https://pingcap.com/docs/v3.0/how-to/deploy/hardware-recommendations/).
+To deploy a multi-master, highly available cluster, see [Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/).
 
-+ The server's disk, memory and CPU choices depend on the capacity planning of the cluster and the deployment topology. It is recommended to deploy three master nodes, three etcd nodes, and several worker nodes to ensure high availability of the online Kubernetes cluster.
+The configuration of the Kubernetes master depends on the number of nodes. The more nodes there are, the more resources are needed. The number of nodes can be adjusted according to your needs.
 
-  Meanwhile, the master node often acts as a worker node (that is, load can also be scheduled to the master node) to make full use of resources. You can set [reserved resources](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/) by kubelet to ensure that the system processes on the machine and the core processes of Kubernetes have sufficient resources to run under high workloads. This ensures the stability of the entire system.
-
-The following text analyzes the deployment plan of three Kubernetes masters, three etcd and several worker nodes. To achieve a highly available deployment of multi-master nodes in Kubernetes, see [Kubernetes official documentation](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/).
+| Nodes in a Kubernetes cluster | Kubernetes master configuration |
+| :--- | :--- |
+| 1-5 | 1vCPUs 4GB Memory|
+| 6-10 | 2vCPUs 8GB Memory|
+| 11-100 | 4vCPUs 16GB Memory|
+| 101-250 | 8vCPUs 32GB Memory|
+| 251-500 | 16vCPUs 64GB Memory|
+| 501-5000 | 32vCPUs 128GB Memory|
 
-## Kubernetes requirements for system resources
+After Kubelet is installed, take the following steps:
 
-- It is required on each machine to have a relatively large SAS disk (at least 1T) to store the data directories of Docker and kubelet.
+1. Save the Kubelet data to a separate disk (it can share a disk with Docker). The data mainly contains the data used by [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). To implement this, set the `--root-dir` parameter:
```

Comment:
```suggestion
1. Save the Kubelet data to a separate disk (it can share the same disk with Docker). The data mainly contains the data used by [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). To implement this, set the `--root-dir` parameter:
```